extensions/open-in-shopify-admin/src/open-in-shopify-admin.ts

```diff
@@ -0,0 +1,33 @@
+import { closeMainWindow, getFrontmostApplication, showToast, Toast } from "@raycast/api";
+import { getBrowserURL, openURL } from "./utils";
+
+const command = async () => {
+  await closeMainWindow();
+  await showToast({ title: "Opening in Shopify admin...", style: Toast.Style.Animated });
+
+  const frontmostApp = await getFrontmostApplication();
+
+  if (frontmostApp.bundleId !== "com.apple.finder") {
+    const url = await getBrowserURL(frontmostApp.name);
+
+    if (!url) {
+      await showToast({ title: "You don't have any website open", style: Toast.Style.Failure });
+      return;
+    }
+
+    const urlParts = url.split("/");
+
+    if (urlParts.length < 2) {
```

Comment:
Can shopify URLs only have a specific length?

Reply:
You are right. This check is not reliable. I have removed it.